fs/rc/rcserver/rcserver.go

```diff
@@ -91,7 +92,7 @@ func (s *Server) Serve() error {
 		// Add username, password into the URL if they are set
 		user, pass := s.opt.HTTPOptions.BasicUser, s.opt.HTTPOptions.BasicPass
 		if user != "" && pass != "" {
-			openURL.User = url.UserPassword(user, pass)
```

Comment:
I think we still want this line don't we?

Reply:
We would not require that anymore.

Reply:
Removing it potentially causes backwards compatibility problems.

That said I don't know of anyone else using it!

Reply:
Okay. I will check against forward compatiblity.

Reply:
Done.